src/runtime/schema/schema.ts

```diff
@@ -40,8 +40,19 @@ export function createLazyState(): LazyState {
 export interface Request extends HTTP.IncomingMessage {
   log: NexusLogger.Logger
 }
+export interface Response extends HTTP.ServerResponse {}
 
-export type ContextContributor = (req: Request) => MaybePromise<Record<string, unknown>>
+export type ContextParams = {
+  /**
+   * Incoming HTTP request
+   */
+  req: Request
+  /**
+   * Server response
+   */
+  res: Response
+}
+export type ContextContributor = (params: ContextParams) => MaybePromise<Record<string, unknown>>
```

Comment:
Wonder about calling this `ContextAdder`?

Also I suggest renaming `ContextParams` to `ContextContributorLens`. Or, `ContextAdderLens` if we go with the rename idea above.

While "lens" is an arbitrary its "lens into the system" meaning is why we use it for the nexus and nexus schema plugin systems. Think it works here too. The nice thing is how distinct it is. Thoughts?